homeassistant/components/fully_kiosk/strings.json

```diff
@@ -69,6 +69,9 @@
       },
       "load_start_url": {
         "name": "Load start URL"
+      },
+      "clear_cache": {
+        "name": "Clear cache of Fully browser"
```

Comment:
I think we can simplify the translation to just `Clear cache` or `Clear browser cache`